### PR TITLE
idk

### PR DESCRIPTION
**Pull request by rastguelenianrn**
i think im doing this right

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -138,4 +145,28 @@ def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
     for player in players:
       for _ in range(numCards):
         player.addCard(deck.getCard())
-    return True
\ No newline at end of file
+    return True
+    print(player)
```
Comment (Lhris) [inline source/CardGames.py]: Make sure to take out this print(player) in the final function if this is for testing purposes

File: source/CardGames.py [hunk 2]
```diff
@@ -138,4 +145,28 @@ def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
...
+deck1 = Deck()
+player1 = Player("john")
+player2 = Player("jeff")
+player3 = Player("george")
+dealer1 = Dealer()
+
+
+dealer1.dealCards(2, [player1], deck1)
+dealer1.dealCards(3, [player2], deck1)
+dealer1.dealCards(4, [player3], deck1)
```
Comment (Lhris) [inline source/CardGames.py]: I understand the multiple lines if you are giving each player a different number of cards, but if this is for testing purposes, you could also make a list of the players and give them an equal number of cards to condense this to one line

File: source/CardGames.py [hunk 3]
```diff
@@ -138,4 +145,28 @@ def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
...
+dealer1.printAllPlayerCards()
```
Comment (Lhris) [inline source/CardGames.py]: Testing should be done in a separate file for PyTest and not in the main CardGames.py

File: test/test_cardgames.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  deck1 = Deck()
+  player1 = Player("john")
+  player2 = Player("jeff")
+  player3 = Player("george")
+  dealer1 = Dealer()
+
+
+  dealer1.dealCards(2, [player1], deck1)
+  dealer1.dealCards(3, [player2], deck1)
+  dealer1.dealCards(4, [player3], deck1)
```
Comment (Lhris) [inline test/test_cardgames.py:14]: If you want, you could condense this into one line by making a list of players (if you are okay with giving each player an equal amount of cards

File: test/test_cardgames.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+from testing_base import *
+
```
Comment (Lhris) [inline test/test_cardgames.py:2]: test_cardgames.py is not super descriptive of what this is testing, you could name it something like test_printCards.py or something along those lines.

**Review by Lhris — COMMENTED** [review]
Added some comments!

File: test/test_cardgames.py (hunk 5 above)
Comment (rastguelenianrn) [reply test/test_cardgames.py:2]: Thanks, I adjusted my code based on all your changes, I believe they are included in my next commits, but there is one file that I am still trying to upload...my bash terminal thinks it has been deleted so I am working that out

**Comment by psk44** [comment]
the code seems to look good man. Good job!

**Review by TMUProfLee — APPROVED** [review]

**Pull request merged**